test/ptc/e2e/system_test.clj

```diff
@@ -47,7 +42,7 @@
         chipwell-barcode (get-in message [::jms/Properties :payload :workflow :chipWellBarcode])
         workflow         (get-in message [::jms/Properties :payload :workflow])
         cloud-prefix     (jms/cloud-prefix bucket workflow)]
-    (jms-tools/queue-messages 1 environment message)
+    (jms-tools/queue-messages 1 queue-environment message)
```

Comment:
I thought purpose of this ticket was to remove the environment passed to this function!?

Reply:
I thought it would be overkill to specify the queue-related values as well...but considering how we recently had to change the prod test queue I see your point 🙃

Reply:
Fixed it!